build/ansible/pmm2/post-build-actions.yml

```diff
@@ -9,9 +9,13 @@
     pmm_client_repos_final: "pmm2-client release"
 
   tasks:
+    - name: Set fact if running inside Docker
+      set_fact:
+        running_in_docker: "{{ lookup('env', 'RUNNING_IN_DOCKER') | default(false) }}"
```

Comment:
```suggestion
        running_in_docker: "{{ lookup('env', 'PMM_RUNNING_IN_DOCKER') | default(false) }}"
```

Reply:
Looking a bit forward, we need to make it compliant with our naming standards in v3, even though v2 doesn't follow them.